crypto/stock-exchange/README.md

```diff
@@ -19,6 +19,7 @@
 
 ```
 {
+  "stockID": <id_of_the_stock>
```

Comment:
I guess we're now passing `stockId`, right?

Reply:
Yes this needs to be changed, will do it now.